Makefile

```diff
@@ -62,9 +62,18 @@ oci-validate-json: validate.go
 oci-validate-examples: cmd/oci-validate-examples/main.go
 	go build ./cmd/oci-validate-examples
```

Comment:
Not really. This tool extracts the examples from the spec for validation. Do we intend to replace the markdown extraction?